janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphIndexTest.java

```diff
@@ -1810,7 +1811,22 @@ private void testCollection(Cardinality cardinality, String property, Object val
             assertEquals(v1, getOnlyElement(graph.query().has(property, Geo.WITHIN, Geoshape.circle(2.0, 2.0, 0.1)).vertices()));
         }
 
-
+        
+        // Test traversal property drop Issue #408
```

Comment:
`testCollection()` is a common method for any (list,set) collection tests. It is called by `testIndexing()` which is called by `@Test testListIndexing()` and `@Test testSetIndexing()`. So I think of `testCollection()` as the gamut of all indexing tests on a collection.  That said, it could be pulled out into its own method `testMultiplePropertyDrop()` and add that to `testIndexing()` after the `testCollection()` calls.  But then do we call that for each property type or just one?  What's your preference?